cmd/main.go

```diff
@@ -56,7 +56,7 @@ func main() {
 
 // startup and shutdown logic
 func run() error {
-	cfg, err := config.LoadConfig(config.DefaultConfigPath)
+	cfg, err := config.LoadConfig(os.Getenv("SSI_HOME"))
```

Comment:
why is this preferable over a configuration-driven path?

Reply:
Because in the future there will be more ways to use. now we deploy as a docker image, pull source, build and launch configure.toml from source code. we also can create release binary, event a Linux package. considering the situation, a well organized path is necessary like other application (service)  do(nginx, ssh,etc..).

Reply:
whit environment var, we can switch home path easily like command line option way which `github.com/ardanlabs/conf` provided.
but `github.com/ardanlabs/conf`'s marshaller can not marshaling toml comments. considering it's important to create a new, default configure file with comments, i use `github.com/suutaku/nice-config` instead.

Reply:
i think it's important to take a hybrid approach
meaning we should define a precedence and overrides
1. default config
2. config file
3. env vars
what do you think?